src/components/organisms/HelpCenter/HelpCenter.tsx

```diff
@@ -17,26 +20,38 @@ export const HelpCenter: React.FC = () => {
     hide: stopLoading,
   } = useShowHide(false);
 
+  const [isShown, toggleShown] = useToggle(false);
+
   const handleHelpClicked = useCallback(
     (url: string) => {
       startLoading();
       setUrl(url);
+      toggleShown();
     },
-    [startLoading]
+    [startLoading, toggleShown]
   );
 
   return (
     <>
-      {url ? (
+      {isShown && url ? (
```

Comment:
looks like it was changed because of this comment. 
https://github.com/sparkletown/sparkle/pull/2253#discussion_r702944849

Let's go back to your initial approach, please. I've created a vote. And looks like more engineers support your way of implementing that.
https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1631272376185900

sorry, that you have to rewrite that again. 🙈

Reply:
> Let's go back to your initial approach, please. I've created a vote. And looks like more engineers support your way of implementing that.
☝️ 
Or maybe just put that URL in a constant and never change it? Not really important to think about this after BM.

Either way, a third way, code will still look out of place. So I guess we can go with @dmitrii-zolotukhin's instinct, whichever functioning way is good enough. Let's just merge it.

Reply:
fixed in a056262f225a39ebd2ee1419e17af080897cc274